OpenSim/Simulation/Model/ScapulothoracicJoint.cpp

```diff
@@ -32,18 +32,17 @@
 //=============================================================================
```

Comment:
This commit contains the only changes from upstream. Effectively, it just tries to clean up the implementation. Main contention points are going to be whether my "cleanups" are cleanups, or just a smattering of Adam-isms. Rather than back-and-forthing, I'll just use the GH review system to explain what's going on.